sonar-db/src/main/java/org/sonar/db/component/ComponentDao.java

```diff
@@ -345,8 +345,16 @@ public void insert(DbSession session, ComponentDto item, ComponentDto... others)
     insert(session, Lists.asList(item, others));
   }
 
-  public void update(DbSession session, ComponentDto item) {
-    mapper(session).update(item);
+  public void update(DbSession session, ComponentUpdateDto component) {
+    mapper(session).update(component);
+  }
+
+  public void applyBChangesForRootComponentUuid(DbSession session, String projectUuid) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUndocumentedApi)